kod.go

```diff
@@ -0,0 +1,62 @@
+package ntp
```

Comment:
I don't think this needs to be in a separate file.  Can you move it to `ntp.go`?